torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -731,6 +731,9 @@ bool LoopNest::computeInline(StmtPtr s) {
 bool LoopNest::computeInline(BufPtr b) {
   // If buf is used or defined in an ExternalCall, we cannot inline it
   auto buf_load_store_uses = findLoadOrStoreUses(root_stmt_);
+  if (!buf_load_store_uses.count(b)) {
```

Comment:
When does this case happen? Just curious. Also, is this relevant / required for this PR?

Reply:
Yes, this is required, and it happens in one of our tests where we concat empty tensors. Before I added simplifier, we had a stmt like
```
{
for (i = 0; i < 0; i++)
  a[i] = 0
}
```
With simplifier the entire block becomes empty.